client/dashboard/sites/settings/site-actions.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMutation } from '@tanstack/react-query';
+import { __experimentalHeading as Heading, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { restoreSitePlanSoftwareMutation } from '../../app/queries';
+import { ActionList } from '../../components/action-list';
+import { DotcomFeatures } from '../../data/constants';
+import type { Site } from '../../data/types';
+
+const useRestorePlanSoftware = ( { slug, options }: Site ) => {
+	const mutation = useMutation( restoreSitePlanSoftwareMutation( slug ) );
+
+	if ( ! options?.is_wpcom_atomic ) {
+		return null;
+	}
+
+	return {
+		title: __( 'Re-install plugins & themes' ),
+		description: __(
+			'If your website is missing plugins and themes that come with your plan you can re-install them here.'
+		),
+		actions: (
+			<Button
+				variant="secondary"
+				size="compact"
+				isBusy={ mutation.isPending }
+				onClick={ () => mutation.mutate() }
+			>
+				{ __( 'Restore' ) }
+			</Button>
+		),
+	};
+};
+
+const useDuplicateSite = ( { capabilities, is_wpcom_staging_site, plan, slug }: Site ) => {
+	if (
+		! (
+			capabilities.manage_options &&
```

Comment:
Is this meant to map with this site owner logic in v1?

https://github.com/Automattic/wp-calypso/blob/d9ed68c65f56ada69441dc6d9790119836a79f55/client/landing/stepper/hooks/use-site-copy.tsx#L77

Are those equivalent checks?

Reply:
I followed the logics [here](https://github.com/Automattic/wp-calypso/blob/a593e382a487d3a477a7cc8b28e1dc0e73e92dcd/client/sites/components/sites-dataviews/actions.tsx#L150-L160) as much as possible.

Reply:
Hmm... it's so weird that we have two different logic for the same thing 😭 we should follow-up with refactoring the checks in these places...

Reply:
🥲